entries/c/cookieyes.com.json

```
@@ -0,0 +1,12 @@
+{
+  "CookieYes": {
+    "domain": "cookieyes.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.cookieyes.com/documentation/two-factor-authentication/",
+    "categories": [
+      "other"
```

I think legal would be a fitting category here.
```suggestion
      "legal"
```

---

I understand! 😄